content/learn/book/getting-started/ecs/_index.md

```diff
@@ -28,7 +28,7 @@ Bevy ECS is Bevy's implementation of the ECS pattern. Unlike other Rust ECS impl
 
     ```rs
     fn print_position_system(query: Query<&Transform>) {
-        for transform in query.iter() {
+        for transform in &query {
```

Comment:
I believe this was kept intentionally because `.iter()` is more beginner friendly.

Reply:
Definitely worth discussing. But I still think we should still use `&`, given that this is the style we "recommend". We chose to update all of our examples to use this pattern / we encourage it in bevy internals.

Its also "more ergonomic" which will make code examples look cleaner.